storefront/src/pages/product detail/productDetail.js

```diff
@@ -1,23 +1,34 @@
 import { useParams } from 'react-router';
 import { useState, useEffect } from "react"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 const ProductDetail = () => {
 
     const { id } = useParams();
 
     const [product, setProduct] = useState(null)
+	const [comments, setComments] = useState(null)
 
     const getProduct =() => {
         fetch("http://localhost:8080/api/products/"+id).then(res => {
             return res.json()
         }).then((data) => {
-            console.log(data);
             setProduct(data)
         })
     }
 
+	const getComments = () => {
+		fetch("http://localhost:8080/api/ratings?product="+id).then(res => {
```

Comment:
Please help to put the api host name to a constant class to avoid duplicating code and easy to maintain.